### DISCUSSION TITLE
配置后无法打开网站

### DISCUSSION THREAD

**Discussion by katfionn** [1Panel-dev/1Panel#4860] [Q&A]
## 配置环境

- 全新安装的1panel最新版本
- Debian11
- 无其他站点
- 站点为纯静态站点
- DNS由DNSpod解析

## 已排查可能出问题的情况

- 证书过期：未过期
- 端口占用：80未被占用
- 权限不足：权限为777、用户和用户组为1000
- 外链不可访问：已测试有问题的外链都是跳往同个站点的另一个页面
- DNS解析有问题：同域名、同配置在另一台服务器使用webmin的apache2模块，是可以成功开启站点、并访问所有页面

## 问题说明

1. 刚开始配置网站后，不添加证书,，无法访问
    ![image](https://github.com/1Panel-dev/1Panel/assets/136874302/1f98d4c9-5cf4-4997-8649-856130ba699d)
2. 添加证书后可以访问index页面，但点击任意一个外链(同站的)，跳过去之后开始出现第1点报错描述的报错、接着所有页面(包括index)都会出现这个报错
3. 此时再关闭https，也是会继续出现该报错
4. 未操作任何东西、等待数分钟后，所有页面都恢复了访问

## 配置截图

- 站点配置
![image](https://github.com/1Panel-dev/1Panel/assets/136874302/f29f102d-1c92-4f04-bf9f-625f52adc30d)
![image](https://github.com/1Panel-dev/1Panel/assets/136874302/f9fd4032-7665-445d-9bfe-45c9600e3f53)
![image](https://github.com/1Panel-dev/1Panel/assets/136874302/86e5c926-790a-4a65-a9d2-e7f151cb451b)
证书是泛域名的，除上图以外，其他都是默认配置

- WAF配置

全局设置，WAF没有黑名单，其他都是默认设置
站点设置，取消了频率访问设置，其他都是默认设置

## 提问的结果

1. 想了解何种原因导致该问题
2. 此后如何避免再次产生该问题

(ps:issue界面搜索“ERR_HTTP2_PROTOCOL_ERROR”关键词，无结果，所以提问)

**Comment by zhengkunwang223** [comment]
可能是被 WAF 拦截了
另外 论坛里也可以搜一下 
https://bbs.fit2cloud.com/c/1p/7

**Reply by katfionn** [reply]
好，感谢大佬回复，我摸索下
